tests/msg_recorder_test.cc

```diff
@@ -18,6 +18,16 @@ using channel_subid_t = cris::core::CRMessageBase::channel_subid_t;
 
 namespace cris::core {
 
+std::filesystem::path GetFirstSubdir(const std::filesystem::path& dir) {
```

Comment:
Are there more reliable ways? "First" is not well-defined so it is error -prone. Also the directory you return maybe some user-created directory paths.